invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/InvitationsMembersModal.js

```diff
@@ -63,12 +99,36 @@ export class InvitationsMembersModal extends Component {
           key="members-users"
           as={Container}
         >
-          <MembersWithRoleSelection
+          <SearchWithRoleSelection
             key="members-users"
             roleOptions={userRoles}
             modalClose={this.handleCloseModal}
             action={api.createInvite}
             onSuccessCallback={this.onMemberSuccess}
+            searchBarTitle={<label>{i18next.t("Member")}</label>}
+            searchBarTooltip={i18next.t(
+              'Users must set profile visibility to "Public" in order to be invited to a community.'
```

Comment:
I think we should be a little bit more vague, something like:
```suggestion
              'Search for users to invite (only users with a public profile can be invited).'
```